config/webpack.js

```diff
@@ -23,6 +23,10 @@ module.exports.webpack = {
           {
             from: './assets/img',
             to: path.resolve(__dirname, '..', '.tmp', 'public', 'img')
+          },
+          {
+            from: './assets/js/jSignature.min.js',
+            to: path.resolve(__dirname, '..', '.tmp', 'public', 'build', 'assets', 'js')
```

Comment:
This library suggests pulling down the files for manual install, so this is why this is harder than using another JS library that either has an NPM package or has a reliable/known CDN.